one line per IP address in `/etc/hosts` is not allowed

<!-- If this is a regression, please open a regression report instead. -->

## Checklist

<!-- Do NOT open an issue until you have: --> 

* [x] Double-checked my configuration.
* [x] Tested using the latest version.
* [x] Used the Emissary executor.

## Summary

What happened/what you expected to happen?

`/etc/hosts` can have a format with one line per IP address:

```
127.0.0.1 dex minio postgres mysql
```

but currently the format isn't allowed in Makefile.

```
make start API=true UI=true
...
# Check dex, minio, postgres and mysql are in hosts file
grep '127.0.0.1[[:blank:]]*minio' /etc/hosts
make: *** [Makefile:455: start] Error 1
```

What version are you running?

master

## Diagnostics

No workflow is needed to reproduce

---
<!-- Issue Author: Don't delete this message to encourage other users to support your issue! -->
**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

--- a/Makefile
+++ b/Makefile
@@ -462,11 +462,11 @@ ifneq ($(PLUGINS),true)
 endif
 	# Check dex, minio, postgres and mysql are in hosts file
 ifeq ($(AUTH_MODE),sso)
-	grep '127.0.0.1[[:blank:]]*dex' /etc/hosts
+	grep '127.0.0.1.*dex' /etc/hosts
 endif
-	grep '127.0.0.1[[:blank:]]*minio' /etc/hosts
-	grep '127.0.0.1[[:blank:]]*postgres' /etc/hosts
-	grep '127.0.0.1[[:blank:]]*mysql' /etc/hosts
+	grep '127.0.0.1.*minio' /etc/hosts
+	grep '127.0.0.1.*postgres' /etc/hosts
+	grep '127.0.0.1.*mysql' /etc/hosts
 	./hack/port-forward.sh
 ifeq ($(RUN_MODE),local)
 	env DEFAULT_REQUEUE_TIME=$(DEFAULT_REQUEUE_TIME) SECURE=$(SECURE) ALWAYS_OFFLOAD_NODE_STATUS=$(ALWAYS_OFFLOAD_NODE_STATUS) LOG_LEVEL=$(LOG_LEVEL) UPPERIO_DB_DEBUG=$(UPPERIO_DB_DEBUG) IMAGE_NAMESPACE=$(IMAGE_NAMESPACE) VERSION=$(VERSION) AUTH_MODE=$(AUTH_MODE) NAMESPACED=$(NAMESPACED) NAMESPACE=$(KUBE_NAMESPACE) MANAGED_NAMESPACE=$(MANAGED_NAMESPACE) UI=$(UI) API=$(API) PLUGINS=$(PLUGINS) $(GOPATH)/bin/goreman -set-ports=false -logtime=false start $(shell if [ -z $GREP_LOGS ]; then echo; else echo "| grep \"$(GREP_LOGS)\""; fi)

